openedx/features/course_duration_limits/access.py

```diff
@@ -113,12 +108,6 @@ def check_course_expired(user, course):
     return ACCESS_GRANTED
 
 
-def get_date_string():
-    # Creating this method to allow unit testing an issue where this string was missing the unicode prefix
-    return '<span class="localized-datetime" data-format="shortDate" data-timezone="{user_timezone}" \
-        data-datetime="{formatted_date}" data-language="{language}">{formatted_date_localized}</span>'
```

Comment:
The test this was designed to support was just testing that intermixing unicode and bytes wouldn't happen in python2. Less of a concern these days.